### PR TITLE
Adds example config for Sync Gateway in distributed index mode.

### LINKED ISSUE #2373: Add sample config for SG running with Accel
The current SG accel config example is an sg_accel config.  We need a SG config for nodes running in an sg_accel-enabled cluster.

### PR DESCRIPTION
**Pull request by ajres**
fixes #2373

### CODE REVIEW

File: examples/basic_sg_distributed_index_config.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "log":[
+    "HTTP+"
+  ],
+  "adminInterface": "127.0.0.1:4985",
+  "interface":"0.0.0.0:4984",
+  "databases":{
+    "default":{
+      "feed_type":"DCPSHARD",
```
Comment (adamcfraser) [inline examples/basic_sg_distributed_index_config.json]: Feed type shouldn't be specified.  (It's not used for SG nodes when running w/ accel).

File: examples/basic_sg_distributed_index_config.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      "server":"http://localhost:8091",
+      "bucket":"default",
+      "channel_index":{
+        "server":"http://localhost:8091",
+        "bucket":"channel_bucket",
+        "writer":false
```
Comment (adamcfraser) [inline examples/basic_sg_distributed_index_config.json]: writer=false can be omitted (it's the default value)

File: examples/basic_sg_distributed_index_config.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      }
+    }
+  },
+  "cluster_config": {
+    "server":"http://localhost:8091",
+    "bucket":"default",
+    "data_dir":"."
+  }
```
Comment (adamcfraser) [inline examples/basic_sg_distributed_index_config.json]: cluster_config is an accel only property - should be omitted on SG nodes.

File: examples/basic_sg_distributed_index_config.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (adamcfraser) [inline examples/basic_sg_distributed_index_config.json]: I don't think we expect anyone external to know what 'distributed index' means - a better name might be something like 'sync_gateway_with_accel_config.json'

**Review by adamcfraser — CHANGES_REQUESTED** [review]

File: examples/basic_sg_distributed_index_config.json (hunk 3 above)
Comment (ajres) [reply examples/basic_sg_distributed_index_config.json]: Removed

File: examples/basic_sg_distributed_index_config.json (hunk 2 above)
Comment (ajres) [reply examples/basic_sg_distributed_index_config.json]: Removed

File: examples/basic_sg_distributed_index_config.json (hunk 1 above)
Comment (ajres) [reply examples/basic_sg_distributed_index_config.json]: Removed

**Pull request merged**